app/src/main/res/layout/activity_main.xml

```diff
@@ -31,11 +32,21 @@
         android:id="@+id/load_btn"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Load Patch"
+        android:text="@string/load_patch"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintHorizontal_bias="0.5"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/click_btn" />
 
+    <Button
+        android:id="@+id/button"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Button"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/errysuprayogi/TinkerSampleApp/blob/a0b231addeed00eb28338fb98a7d40f48abf7015/app/src/main/res/layout/activity_main.xml#L46"></span>Hardcoded string “Button”, should use <code>@string</code> resource</td>
    </tr>
  </tbody>
</table>